static/js/components/ProgramCourse.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import React from 'react';
+import Popover from 'material-ui/Popover';
+
+
+export default class ProgramCourse extends React.Component {
+  props: {
+    id: number,
+    title: string,
+    description: string,
+    url: string,
+    enrollment_text: string,
+  }
+  state = {
+    isOpen: false,
+  }
+  handleClick = (event) => {
+    // This prevents ghost click.
+    event.preventDefault();
+
+    this.setState({
+      isOpen: true,
+      anchorEl: event.currentTarget,
+    });
+  }
+  handleRequestClose = () => {
+    this.setState({
+      isOpen: false,
+    });
+  }
+  render() {
+    const {
+      title,
+      description,
+      url,
+      enrollment_text: enrollmentText
+    } = this.props;
+    const {
+      isOpen,
+      anchorEl
+    } = this.state;
+
+    let titleEl, popoverLink;
+    if (url) {
+      titleEl = <a href={url}>{title}</a>;
```

Comment:
will it be possible to click on `titleEl`? it looks below like it gets wrapped in an `h4` tag which has an `onclick` handler